mod_modular_vanilla/hooks/entity/tactical/player.nut

```diff
@@ -109,4 +109,52 @@
 			this.fillAttributeLevelUpValues(::Const.XP.MaxLevelWithPerkpoints - 1);
 		}
 	}}.setStartValuesEx;
+
+	q.setMoraleState = @() { function setMoraleState( _m )
+	{
+		if (this.getCurrentProperties().MV_ForbiddenMoraleStates.find(_m) != null)
```

Comment:
This check should probably also exist in the regular actor.nut. 
Even now traits, like insecure, could theoretically be put on non-player by mods. And future NPC effects might wanna make use of the forbidden morale states system more regularly